test/quantization/test_qat_module.py

```diff
@@ -82,6 +270,7 @@ def test_conv_bn_relu(
 
             cls = ConvBnReLU2d if use_relu else ConvBn2d
             qat_op = cls(
+                copy.deepcopy(bn_op),
```

Comment:
can you move bn after the conv2d parameters

Reply:
one limitation is that some of the conv params are optional, and all required params have to come before optional params.  I thought this was worth changing the arg order (so we can make bn required), but flexible - lmk.

Reply:
OK

Reply:
actually, I take it back, passing the module instead of params leads to other problems.  Will switch back to params and move it after conv